### ISSUE TITLE
Track modeling results in a database

### ISSUE THREAD

**Issue by dmarx** [dmarx/make_for_datascience#5]
For each run of the test target, log:

1. The date
2. Which models were run
3. The outcomes of all generated reports
4. The commit id for the HEAD

Schema outline:

* EXPERIMENTS
  * experiment_id
  * model_name 
  * commit id

* RESULTS
  * result_id
  * experiment_id
  * result_name  

* RESULTS_DATA (melted table)
  * datum_id
  * result_id
  * field_id
  * field_value

* RESULTS_FIELDS
  * field_id
  * field_name

**Comment by dmarx** [comment]
Works for bootstrap, schema doesn't accomodate confusion matrix particularly well. Probably need to re-write the schema. Otherwise, db added and working in  commit 3e7b8a5

**Comment by dmarx** [comment]
We can assume results aren't more complex than tables. Instead of (key, val), let's do (row, col, val). Boom.

**Comment by dmarx** [comment]
e6685a3

**Comment by dmarx** [comment]
Interestingly, I magically lost dbapi.r at some point. I'm pretty sure it was when I rebased eval_db on top of master. I ended up rewriting the script, but from referencing the comment above we're actually able to access it in this commit that appears to have been orphaned or something: 3e7b8a5

Need to do some forensics to figure out what happened with that commit and make sure it doesn't happen again in the future.
